### DISCUSSION TITLE
Announcing Nivenly topics and discussions on Hachyderm

### DISCUSSION THREAD

**Discussion by wesen** [nivenly/community#3] [Ideas]
[hachyderm](https://hachyderm.io) is a nivenly project and the two are often used interchangeably (at least in hachyderm discourse). They also share a discord server and are still linked to Kris Nova and quintessence as public personas. To increase democratic participation in nivenly discourse and discussions, would it be possible to use hachyderm server notifications and official accounts to maybe run a weekly run-down of things happening around nivenly?

Due to the transitory nature of mastodon (especially with federation issues), server notifications seem like a more stable way to reach the significant portion of nivenly members who are also active on mastodon.

**Comment by nivenly-foundation** [comment]
Hello!

This is actually something that will require some deeper discussion. We definitely agree that there needs to be easier ways for Nivenly Members to receive updates. As an individual project, posting notifications on Hachyderm would actually blur the lines rather than helping to keep them well understood. Basically: the vast majority of Hachydermians are not Nivenly Co-Op Members, but posting Nivenly updates on Hachyderm's notifications would definitely start to give the impression that Hachyderm == Nivenly.

We've been looking for solutions here and would love to meet the community where they look to receive their notifications in a way that still feels informative but not invasive. 

Are there tools or places that you would look for notifications about Nivenly, aside from Nivenly's social media account and blogs? Our first goal would be to create a feedback loop where members are already looking, rather than introduce another new tool  that potentially requires an account, etc.

**Reply by wesen** [reply]
A weekly summary email, like many of the organization I belong to do, would be great. Similarly, individual emails when a "big" topic is on the table. I agree that hachyderm being such a big instance would indeed blur the lines.

**Reply by nivenly-foundation** [reply]
Sounds great! 

Part of building Nivenly is for us to queue up what the Community has indicated they need and what resources are needed to provide it. We'll leave this discussion open as we'll include information about this in our upcoming posts (we have some queued, relevantly about governance and comms is definitely a part of that).

**Reply by MadokVaur** [reply]
OK

Seeing that my reply to this via email hasn't/doesn't propagate back here, repost

(Full disclosure: I am only an egg)

> Basically: the vast majority of Hachydermians are not Nivenly Co-Op Members, 

At some point (probably best sooner than later) this fact will need to be made clear on Hachyderm dot io

Tears will be shed

No sooner did word of Haidra break on Hachyderm than posts were appearing -- not many, granted, but -- to the tune of "Why weren't we told of this!! Is this how Hachyderm governs itself? What about our representation?"

See: Hachyderm != Nivenly


> but posting Nivenly updates on Hachyderm's notifications would definitely start to give the impression that Hachyderm == Nivenly.

So, yeah

I think there should be Nivenly Foundation posts *to* Hachyderm, but it should be limited to maybe a daily informational boilerplate that briefly lays out the project-oriented structure of Nivenly (and names them: I'd be willing to bet that 99% of Hachyderm people have never heard of Aurae) and that gently suggests Nivenly is a co-op and that representation comes through General memebership

(Kinda like the REI and Puget Consumers Co-ops I joined when I first moved to Seattle in the mid-70s. I could shop at the stores but I didn't get any annual dividend if I wasn't a member)

*Not* a day-by-day breaking news recap though, although maybe containing a link to the latest activity, elsewhere

Finally, because I think this comes up in another post, I'd probably vote for the main discussion forum to be on Github (which this is replying to) than on Discord

Discord has too much noise, particularly because of what I follow there, but on Github the discussion would take place explicitly through the Github Nivenly community/discussions

**Reply by nivenly-foundation** [reply]
Hello!

Thank you for taking the time to write out all your thoughts! (Or even re-write them, if we're reading one of your comments correctly.)

We definitely agree about clearly communicating that Hachyderm =/= Nivenly. This is something we've mentioned before, and expect that we'll need to continue to mention as updates progress. In the blog post we're working on now, focused on governance and engagement for the Haidra discussions, we do include an explainer about membership including directly addressing this.

For newsletter updates, in general: we wholeheartedly agree! Since everyone working on Nivenly is also working on dayjobs, the cadence of updates is probably less weekly and more monthly (that's really an implementation detail, just wanted to mention).

Once we discuss how members want to receive updates, like a member email, we'll mostly need a volunteer to help maintain it. They'd have visibility into what they needed to write and send out the newsletter, but as we're still running lean we'd need to expand our resources to ensure that it'd be delivered regularly and with the appropriate information.

**Reply by nivenly-foundation** [reply]
Oh, and also to explicitly state: Discord is definitely best suited for Q&A and other discourse style engagement, and probably isn't the best place for a "co-op member report" in terms of readability and visibility, agreed.
